### DISCUSSION TITLE
河南大学

### DISCUSSION THREAD

**Discussion by Maxwell-Niu** [CollegesChat/university-information#235] [Ideas]
环境怎样

**Comment by Waj30** [comment]
学校环境还行，宿舍极差

**Comment by JOYhappy888** [comment]
金明校区食堂荤素不均，有些菜会引起腹泻。宿舍自来水的水质不太好，每层楼饮水机打热水，顶楼会漏雨。但教室环境不错，桌椅较宽。图书馆引入AI导览，新翻修，挺漂亮。

总之开封校区的吃和住宿条件不太好，龙子湖校区则无住宿担忧。

**Comment by PaperWr** [comment]
请问河大有早晚自习吗？校园跑之类的可以骑车吗？

**Comment by Waj30** [comment]
没有早晚自习，但是可能晚上会有课。开封校区校园跑必须在操场进行，不允许骑车
&nbsp;


3089555484@@
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2025年6月22日(星期天) 中午1:14
收件人: ***@***.***&gt;; 
抄送: "3089555484@@***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [CollegesChat/university-information] 河南大学 (Discussion #235)















请问河大有早晚自习吗？校园跑之类的可以骑车吗？

-
直接回复这封邮件，在GitHub上查看它，或者取消订阅.
***@***.*** &gt;

**Comment by 2178079525** [comment]
转专业容易吗

**Reply by gowithwind1551** [reply]
每个院不一样，但好像都有成绩要求。
